packages/atomic/src/components/common/atomic-result/atomic-result.tsx

```diff
@@ -4,22 +4,22 @@ import {applyFocusVisiblePolyfill} from '../../../utils/initialization-utils';
 import {
   DisplayConfig,
   ResultContextEvent,
-} from '../result-template-components/result-template-decorators';
-import {ResultRenderingFunction} from '../result-lists/result-list-common';
-import {AtomicStore} from '../atomic-search-interface/store';
+} from '../../search/result-template-components/result-template-decorators';
+import {ResultRenderingFunction} from '../result-list/result-list-common';
 import {
   ResultLayout,
   ResultDisplayDensity,
   ResultDisplayImageSize,
   ResultDisplayLayout,
-} from '../../common/layout/display-options';
+} from '../layout/display-options';
+import {AtomicCommonStore, AtomicCommonStoreData} from '../interface/store';
 
 /**
  * The `atomic-result` component is used internally by the `atomic-result-list` component.
  */
 @Component({
   tag: 'atomic-result',
-  styleUrl: '../../common/result-styles/result.pcss',
+  styleUrl: 'atomic-result.pcss',
```

Comment:
I'm not sure if the code for the result list is going to stay the same for every implementation, I think it would be safer not to re-use it like this and to instead add an `atomic-recs-result` component just like we did for insight.

Reply:
Okay I'll move it back to search, but I'll create the new component in an upcoming PR